### PR TITLE
Cura: update to 4.3.0

### PR DESCRIPTION
**Pull request by karl-nilsson**
Preparing for the next Cura release

Also take ownership of libSavitar

### CODE REVIEW

**Comment by karl-nilsson** [comment]
Built and tested on x64 glibc
Testers welcome!

File: srcpkgs/cura-fdm-materials/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'cura-fdm-materials'
 pkgname=cura-fdm-materials
-version=2019.08.21
+version=4.3.0
```
Comment (abenson) [inline srcpkgs/cura-fdm-materials/template]: This won't update, needs a revert

Comment (abenson) [reply srcpkgs/cura-fdm-materials/template]: `reverts="2019.08.21_1"`

I think that's right.

**Comment by karl-nilsson** [comment]
@abenson fixed

**Pull request merged**